### DISCUSSION TITLE
Didnt received a Starstruck Badge

### DISCUSSION THREAD

**Discussion by havelhakimi** [Schweinepriester/github-profile-achievements#950] [General]
I have 18 stars on this [repo](https://github.com/havelhakimi/TLA) but still didn't received a Starstruck Badge. Is there something else in the criterion that i didn't met yet ?

**Comment by omyto** [comment]
You’ve got it! https://github.com/havelhakimi?tab=achievements
Just go to [Settings](https://github.com/settings/profile) and enable the option to show achievements on your profile.
![image](https://github.com/user-attachments/assets/cd624c8d-1bfd-4f9b-b07e-bbd004fac92b)

**Reply by AyushKatre05** [reply]
https://github.com/AyushKatre05/AyushKatre05.git star my repo ui will star yours

**Comment by RoshansaiK** [comment]
You need at least 16 stars on public repositories to earn the Starstruck badge; ensure your starred repo is public to meet the criteria.

**Reply by AyushKatre05** [reply]
https://github.com/AyushKatre05/AyushKatre05.git star my repo ui will star yours

**Comment by FahimWayez** [comment]
Hi, this seems to be a bug with GitHub. I think you should [contact GitHub support](https://support.github.com/) and ask them about the issue. I have seen others with the similar problem, and they did the same - which fixed the issue.

**Reply by AyushKatre05** [reply]
https://github.com/AyushKatre05/AyushKatre05.git star my repo ui will star yours

**Comment by nanisadw3** [comment]
Hey, this looks like a potential bug with GitHub. I’d recommend reaching out to GitHub support to check on the issue. I’ve seen others run into a similar problem, and contacting support helped them resolve it. Hope it gets sorted out soon!

**Reply by AyushKatre05** [reply]
https://github.com/AyushKatre05/AyushKatre05.git star my repo ui will star yours
